File: app/src/main/java/com/ikem/nwodo/cryptonyte/MyYAxisValueFormatter.java
package com.ikem.nwodo.cryptonyte;

import com.github.mikephil.charting.components.AxisBase;
import com.github.mikephil.charting.formatter.IAxisValueFormatter;

public class MyYAxisValueFormatter implements IAxisValueFormatter {

    @Override
    public String getFormattedValue(float value, AxisBase axis) {
        return "$" + NumberFormatter.graphYAxisFormatter(value);
    }
}

File: shared/src/main/java/com/ikem/shared/utils/graph/MyYAxisValueFormatter.kt
package com.ikem.nwodo.cryptonyte.utils.graph

import com.github.mikephil.charting.components.AxisBase
import com.github.mikephil.charting.formatter.IAxisValueFormatter

class MyYAxisValueFormatter : IAxisValueFormatter {

    override fun getFormattedValue(value: Float, axis: AxisBase): String {
        return "$" + NumberFormatter.graphYAxisFormatter(value)
    }
}
